kustomize/single-namespace/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Adds namespace to all resources.
+namespace: influxdb
```

Comment:
My kustomize skills are not very strong, I couldn't figure out a way to have a user customizable namespace without having a several patches that they would need to edit. Is there a better way to do this?

Reply:
IIRC you just omit the namespace and thus you'll end up using whatever the default namespace of the client or the namespace passed with `-n` on kubectl.

Another popular thing that people using kustomize do is that the end users themselves create their little kustomization.yaml file that references ours and sets the namespace

Reply:
So that definitely works for just the generic namespace that is added to every resource. But what about in the `manager.yaml` file where we need to also give that value to `kubit` so it knows which namespace to watch?

Reply:
Option 1: https://kubernetes.io/docs/concepts/workloads/pods/downward-api/

Option 2: make kubit deduce the current namespace the pod is in (possibly that's the default k8s client namespace when using in-cluster config)